src/stories/Popover.stories.tsx

```
@@ -3,7 +3,7 @@ import { useArgs } from '@storybook/preview-api';
 import { Meta, StoryObj } from '@storybook/react';
 import { FiChevronDown } from 'react-icons/fi';
 
-const meta: Meta = {
+const meta: Meta<any> = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)